README.md

```diff
@@ -305,12 +305,14 @@ used to satisfy the following use-cases:
+| `/tiles`   | POST   | Get streamed features, according to hard constraints. Accepts encoding types `text/jsonl` or `application/binary` | List of objects containing `mapId`, `layerId`, `tileIds`, and optional `stringPoolOffsets`.                                                         | `text/jsonl` or `application/binary`                                                                                                                                                                                                                              |
+| `/status`  | GET    | Server status page                                                                                                | None                                                                                                                                                | `text/html`                                                                                                                                                                                                                                                       |
+| `/locate`  | POST   | Obtain a list of tile-layer combinations providing a feature that satisfies given ID field constraints.           | `application/json`: List of external references, where each is a Request object with `mapId`, `typeId` and `featureId` (list of external ID parts). | `application/json`: List of lists of Resolution objects, where each corresponds to the Request object index. Each Resolution object includes `tileId`, `typeId`, and `featureId`.                                                                                 |
+| `/config`  | GET    | Access the config yaml-file content.                                                                              | None                                                                                                                                                | `application/json`: Contains the `sources` and `http-settings` from the config-yaml as a JSON representation. The returned JSON object has a `model`, `schema` and `readOnly` key. The schema is controlled through the `--config-schema` command line parameter. |
+| `/config`  | POST   | Write the config yaml-file content. Enabled iff `--allow-config-access` is passed to mapget.                      | `application/json`                                                                                                                                  | `text/plain` (if an error occurs)                                                                                                                                                                                                                                 |
```

Comment:
Use `--allow-config-post/write` to clearly indicate that this is about writing (and does not affect reading)